bindata/manifests/daemon/daemonset.yaml

```diff
@@ -93,6 +93,7 @@ spec:
           preStop:
             exec:
               command: ["/bindata/scripts/clean-k8s-services.sh"]
+      terminationGracePeriodSeconds: 900
```

Comment:
Do we want to make it configurable?

Reply:
It needs to be at least as high as the longest expected plugin apply + some buffer for safety. Is there a good value for this?